gnomad_hail/resources/resource_utils.py

```diff
@@ -127,23 +234,29 @@ def bm(self) -> BlockMatrix:
         """
         return BlockMatrix.read(self.path)
 
+    def import_resource(self, overwrite: bool = False, **kwargs):
+        """
+        Imports the BlockMatrixResource using its import_func and writes it in its path.
+
+        :param overwrite: If ``True``, overwrite an existing file at the destination.
+        :param kwargs: Any additional parameters to be passed to BlockMatrix.write
+        :return: Nothing
+        """
+        self.import_func(**self.import_sources).write(self.path, overwrite=False, **kwargs)
+
 
 class BaseVersionedResource(BaseResource, ABC):
     """
     Abstract class for a versioned resource
-    """
 
-    def __init__(self, default_version: str, versions: Dict[str, BaseResource]):
-        """
-        Creates a versioned resource.
-        The `path`/`source_path` attributes of the versioned resource are those
-        of the default version of the resource.
-        In addition, all versions of the resource are stored in the `versions` attribute.
+    The `path`/`source_path` attributes of the versioned resource are those of the default version of the resource.
```

Comment:
`source_path` should probably be replaced with `import_sources` here. `source_path` is one of the keys in `import_sources` for most resources, but it is not an attribute of resource objects. Same comment for docstrings on other versioned resource classes.

Reply:
yes! That's definitely a remnant of a previous implementation iteration that wasn't caught earlier!